C/services/north/north.cpp

```diff
@@ -376,6 +375,7 @@ void NorthService::createConfigCategories(DefaultConfigCategory configCategory,
                                                defaultServiceConfig);
 	defConfig += serviceConfig;
 
+	defConfig.setDescription(current_name);	// TODO We do not have access to the description
```

Comment:
Do we have any JIRA for TODO comment "We do not have access to the description"? It would be nice to have so that later/future we will come to know to fix this comment

Reply:
No, that has been around for a very long time and is very low priority as there is no meaningful comment usually.